org.eclipse.jdt.core.javac/src/org/eclipse/jdt/core/dom/JavacCompilationUnitResolver.java

```diff
@@ -404,7 +404,7 @@ private AST createAST(Map<String, String> options, int level, Context context) {
 		long sourceLevel = CompilerOptions.versionToJdkLevel(sourceModeSetting);
 		if (sourceLevel == 0) {
 			// unknown sourceModeSetting
-			sourceLevel = ClassFileConstants.JDK21; // TODO latest
+			sourceLevel = ClassFileConstants.JDK23; // TODO latest
```

Comment:
Isn't there some "latest" we can reference here?

Reply:
Fixed now.